.gitignore

```diff
@@ -118,3 +118,7 @@ conda*.yaml
 *.zip
 .vscode/
 scripts/repo_metrics/config.py
+ml-100k/
+ml-10M100K/
```

Comment:
is this a typo?

Reply:
These are the movielens datasets folders when they are downloaded